.github/workflows/deploy.yml

```
@@ -0,0 +1,35 @@
+        with:
+          node-version: '22'
+          cache: 'pnpm'
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Build (OpenNext + Cloudflare)
+        run: pnpm run cf:build
+
+      - name: Deploy to Cloudflare Pages
+        run: npx wrangler pages deploy --commit-message "${{ github.event.head_commit.message || 'deploy' }}"
```

actionlint (1.7.12)

[error] "github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)